### PR TITLE
Add logging role

### PR DESCRIPTION
**Pull request by loshkovskyi**
The logging role configures Logstash 1.5 with Docker and systemd. Containers with Logstash are started on each host and collect logs from multiple inputs. This allows us to filter all logs and send them to a central location. At this time, we support several outputs such as HDFS and Logentries. By default all logs are sent to STDOUT. More details are available in the [README](https://github.com/CiscoCloud/microservices-infrastructure/tree/feature/logging-role/roles/logging) file.

### CODE REVIEW

**Comment by keithchambers** [comment]
@loshkovskyi awesome! I'll take a look and get this merged today. I might make some smaller edits here and there.

Also I'm going to make a couple edits to https://github.com/CiscoCloud/docker-logstash, add a license, and start building it in our CiscoCloud project on Docker Hub.

Thank you!!

File: roles/logstash/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+logstash_image: loshkovskyi/docker-logstash
```
Comment (stevendborrelli) [inline roles/logstash/defaults/main.yml]: As part of #277, can we move this to the ciscocloud docker hub account?

File: roles/logstash/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+logstash_image_tag: latest
```
Comment (stevendborrelli) [inline roles/logstash/defaults/main.yml]: instead of of latest, let's use a version tag

**Comment by keithchambers** [comment]
@stevendborrelli I agree. Let me get the build in to CiscoCloud Docker Hub.

**Comment by keithchambers** [comment]
@loshkovskyi can you please test these changes and verify everything works as expected? Thx!

**Comment by loshkovskyi** [comment]
Yes, will do. Right now this commit [1ff957d2](https://github.com/CiscoCloud/microservices-infrastructure/commit/1ff957d2af356e857c9758524c475b88f8200276) breaks the role:
```
+    - logstash
     - consul-template
     - docker
```
Logstash must be started after docker.

**Comment by keithchambers** [comment]
Fixed!

**Comment by loshkovskyi** [comment]
@keithchambers , thank you. Everything looks fine.

**Pull request merged**